### ISSUE TITLE
Development/test auto-passing?

### ISSUE THREAD

**Issue by eric-hu** [achiurizo/rack-recaptcha#9]
Does rack-recaptcha have a flag that can be set so that recaptcha's are auto-passing in test environments?  I'd like to include my recaptcha pages in my cucumber tests

**Comment by achiurizo** [comment]
you can now set this by doing

``` ruby
Rack::Recaptcha.test_mode!
```

or for a false return:

``` ruby
 Rack::Recaptcha.test_mode! :return => false
```

This is available with the latest version 0.5.0

**Comment by eric-hu** [comment]
awesome!  Thanks for putting this feature in
